src/compile/model.ts

```diff
@@ -15,6 +15,12 @@ import {VgData, VgMarkGroup, VgScale, VgAxis, VgLegend} from '../vega.schema';
 import {DataComponent} from './data/data';
 import {LayoutComponent} from './layout';
 import {ScaleComponents} from './scale';
+/* tslint:disable:no-unused-variable */
```

Comment:
`tslint` has `no-unused-variables` on, and these variables are, _technically_ unused in the code, _however_, they need to be named so their names can be used in the resulting compiled `.d.ts` (you will get compiler errors from typescript without the references).